--- NaturalDocs-1.35/Modules/NaturalDocs/Settings.pm.orig	2005-04-14 18:22:22.000000000 +0200
+++ NaturalDocs-1.35/Modules/NaturalDocs/Settings.pm	2005-04-14 18:23:06.000000000 +0200
@@ -475,17 +475,17 @@
 # Function: StyleDirectory
 # Returns the main style directory.
 sub StyleDirectory
-    {  return NaturalDocs::File->JoinPaths($FindBin::RealBin, 'Styles', 1);  };
+    {  return NaturalDocs::File->JoinPaths('/usr/share/NaturalDocs', 'Styles', 1);  };
 
 # Function: JavaScriptDirectory
 # Returns the main JavaScript directory.
 sub JavaScriptDirectory
-    {  return NaturalDocs::File->JoinPaths($FindBin::RealBin, 'JavaScript', 1);  };
+    {  return NaturalDocs::File->JoinPaths('/usr/share/NaturalDocs', 'JavaScript', 1);  };
 
 # Function: ConfigDirectory
 # Returns the main configuration directory.
 sub ConfigDirectory
-    {  return NaturalDocs::File->JoinPaths($FindBin::RealBin, 'Config', 1);  };
+    {  return NaturalDocs::File->JoinPaths('/usr/share/NaturalDocs', 'Config', 1);  };
 
 # Function: DocumentedOnly
 # Returns whether undocumented code aspects should be included in the output.
--- NaturalDocs-1.35/NaturalDocs.orig	2005-04-14 18:24:35.000000000 +0200
+++ NaturalDocs-1.35/NaturalDocs	2005-04-14 18:24:55.000000000 +0200
@@ -66,7 +66,6 @@
 use English '-no_match_vars';
 
 use FindBin;
-use lib "$FindBin::RealBin/Modules";
 
 sub INIT
     {
